src/datasources/perf-ds/datasource.ts

```diff
@@ -341,6 +343,29 @@ export class OpenNMSDatasource {
     return Promise.resolve([]);
   }
 
+  metricFindNodesLocation(searchLimit) {
```

Comment:
The pluralization here is even more weird. Should be `metricFindLocations`.

Reply:
Applied change